src/main/java/io/stargate/sgv2/jsonapi/service/resolver/CreateCollectionCommandResolver.java

```diff
@@ -58,6 +59,9 @@ public Class<CreateCollectionCommand> getCommandClass() {
   @Override
   public Operation resolveKeyspaceCommand(
       CommandContext<KeyspaceSchemaObject> ctx, CreateCollectionCommand command) {
+
+    validateSchemaName(command.name(), NamingRules.COLLECTION);
```

Comment:
see comments for changeing validateSchemaName to return the name, then assign the return to a var here and re-use the validated name